### PR TITLE
Implementing Pin row 3d model and refactor footprint plated holes

### PR DESCRIPTION
**Pull request by anas-sarkez**
Screenshot 2024-10-18 at 3 18 29 PM
Screenshot 2024-10-18 at 3 26 22 PM

### CODE REVIEW

**Comment by anas-sarkez** [comment]
#73

**Review by seveibar — APPROVED** [review]
Looking good, can you make the top shorter than the bottom?

im also not sure about the radius of the pin, it looks thick but i could be wrong

**Comment by seveibar** [comment]
Also can you add support for Footprinter3d for wg “pinrow6”? Can be future issue

**Review by abhijitxy — APPROVED** [review]

File: lib/FootprintPlatedHole.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
   if (hole.shape === "circle") {
     return (
-      <Colorize color={[255, 0, 0]}>
+      <Colorize color="#b87333">
         <Translate offset={[hole.x, hole.y, 0]}>
           {/* <Rotate axis="z" angle={90}> */}
-          <Cylinder radius={hole.outer_diameter} height={0.01} />
+          <Subtract>
+            <Cylinder radius={hole.outer_diameter} height={0.01} />
+            <Cylinder radius={hole.hole_diameter} height={0.01} />
```
Comment (seveibar) [inline lib/FootprintPlatedHole.tsx:12]: radius issue

**Comment by anas-sarkez** [comment]
@seveibar 
check last commit changes

**Review by seveibar — APPROVED** [review]

**Pull request merged**